app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -90,6 +107,15 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         that.$somethingIsWrongForm.append($('<input type="hidden" name="referrer">').val(document.referrer || 'unknown'))
       }
 
+      function setHiddenValuesNotUsefulForm () {
+        var currentPathname = window.location.pathname;
+        var encodedPathname = encodeURI(currentPathname);
+        var finalPathName = encodedPathname.replace(/[^\s=\/?&]+(?:@|%40)[^\s=\/?&]+/, '[email]')
```

Comment:
I suggest doing any regex replacement _before_ you encode the path name - probably easier to understand!